interactions/api/http/interaction.py

```diff
@@ -15,22 +15,34 @@ def __init__(self) -> None:
         pass
 
     async def get_application_commands(
-        self, application_id: Union[int, Snowflake], guild_id: Optional[int] = None
+        self,
+        application_id: Union[int, Snowflake],
+        guild_id: Optional[int] = None,
+        with_localizations: Optional[bool] = None,
     ) -> List[dict]:
         """
         Get all application commands from an application.
 
         :param application_id: Application ID snowflake
         :param guild_id: Guild to get commands from, if specified. Defaults to global (None)
+        :param with_localizations: 	Whether to include full localization dictionaries (name_localizations and description_localizations) in the returned objects, instead of the name_localized and description_localized fields. Default false.
```

Comment:
```suggestion
        :param with_localizations: Whether to include full localization dictionaries (name_localizations and description_localizations) in the returned objects, instead of the name_localized and description_localized fields. Default false.
```